app/assets/javascripts/react_router.js

```diff
@@ -123,209 +123,209 @@ class ReactRouter extends React.Component<Props> {
     />
   );
 
+  componentDidCatch(error, info) {
```

Comment:
Should we remove this again as my tests using the error boundary for 401 errors were not successful?